pkg/debugapi/router.go

```diff
@@ -158,6 +158,10 @@ func (s *Service) newRouter() *mux.Router {
 			"GET":  http.HandlerFunc(s.swapCashoutStatusHandler),
 			"POST": http.HandlerFunc(s.swapCashoutHandler),
 		})
+
+		router.Handle("/chequebook/cashout/redo/{peer}", jsonhttp.MethodHandler{
```

Comment:
```suggestion
		router.Handle("/chequebook/redocashout/{peer}", jsonhttp.MethodHandler{
```

Reply:
just in case the path got confused with `{peer}` and `redo`; that could really happen though 😉